src/aiortc/rtcrtpreceiver.py

```diff
@@ -264,8 +264,8 @@ def __init__(self, kind: str, transport: RTCDtlsTransport) -> None:
             self.__nack_generator = None
             self.__remote_bitrate_estimator = None
         elif kind == "keypoints":
-            self.__jitter_buffer = JitterBuffer(capacity=16)
-            self.__nack_generator = None
+            self.__jitter_buffer = JitterBuffer(capacity=128)
```

Comment:
I remember this number was a little tricky. How did you decide to put 128?

Reply:
No good reason - 16 was too small and was causing overwriting of keypoints that were received out of order. 128 may be too large, we could get away with some lower tolerance. But, this felt reasonable to me for now, we can change it if we observe issues again